src/components/Icon/DotsIcon/DotsIcon.tsx

```diff
@@ -4,16 +4,36 @@ import Icon, { IIconProps, propTypes as iconPropTypes } from '../Icon';
 import { lucidClassNames } from '../../../util/style-helpers';
 import { omitProps } from '../../../util/component-types';
 import { Color } from '../Icon';
+import PropTypes from 'react-peek/prop-types';
 
 const cx = lucidClassNames.bind('&-DotsIcon');
 
-interface IDotsIconProps extends IIconProps {}
+const { oneOf } = PropTypes;
+
+enum Orientation {
+	vertical = 'vertical',
+	horizontal = 'horizontal'
+}
+interface IDotsIconProps extends IIconProps {
+	direction?: keyof typeof Orientation;
+}
 
 export const DotsIcon = ({
 	className,
+	direction = Orientation.horizontal,
 	color = Color.primary,
 	...passThroughs
 }: IDotsIconProps) => {
+	const isVerticalOrientation = _.isEqual(direction, Orientation.vertical);
```

Comment:
`_.isEqual` is good for deep comparisons, but here you're just doing a string comparison, so it's probably simpler to do a `===` compare.

Reply:
done